components/tag-list/test/tag-list.vdiff.js

```diff
@@ -36,27 +36,81 @@ describe('tag-list', () => {
 	});
 
 	describe('tag list item style', () => {
+		let elem;
+		afterEach(() => {
+			elem.querySelector('d2l-tag-list-item').resetHasDisplayedKeyboardTooltip();
+		});
+
```

Comment:
This piece feels a bit hacky so let me know if there's a better way. The global var needs to be reset between tests if we want to test that functionality.